sbi/utils/sbiutils.py

```diff
@@ -20,112 +23,115 @@ def forward(self, tensor):
         )
 
-    if context.shape[0] != inputs.shape[0]:
-        # multiple parameters, single observation:
-        # repeat the context to match the parameters
-        context = context.repeat(inputs.shape[0], 1)
+    if x.shape[0] != theta.shape[0]:
+        # multiple parameter sets theta, single observation x:
+        # repeat the x to match the parameters theta
+        x = x.repeat(theta.shape[0], 1)
 
-    if inputs.shape[0] != context.shape[0]:
+    if theta.shape[0] != x.shape[0]:
         # catch all remaining errors after shape-mangling above
         # THIS SHOULD NEVER HAPPEN
-        raise ValueError(
-            "Number of input items must be equal to number of context items."
-        )
+        raise ValueError("Number of theta items must be equal to number of x items.")
 
-    return inputs, context
+    return theta, x
 
 
 def sample_posterior_within_prior(
     posterior_nn: torch.nn.Module,
     prior: torch.distributions.Distribution,
-    context: torch.Tensor,
+    x: torch.Tensor,
     num_samples: int = 1,
     patience: int = 5,
 ) -> Tuple[torch.Tensor, float]:
-    """Return samples from a posterior within the support of the prior via rejection sampling. 
+    r"""Return samples from a posterior $p(\theta|x)$ within the support of the prior
+     via rejection sampling.
     
-    This is relevant for snpe methods and flows for which the posterior tends to have mass outside the prior boundaries. 
+    This is relevant for snpe methods and flows for which the posterior tends to have
+     mass outside the prior boundaries.
     
     This function uses rejection sampling with samples from posterior, to do two things: 
         1) obtain posterior samples within the prior support. 
-        2) calculate the fraction of accepted samples as a proxy for correcting the density during evaluation of the posterior. 
+        2) calculate the fraction of accepted samples as a proxy for correcting the
+         density during evaluation of the posterior.
     
     Arguments:
         posterior_nn {torch.nn.Module} -- neural net representing the posterior
         prior {torch.distributions.Distribution} -- torch distribution prior
-        context {torch.Tensor} -- context for the posterior, i.e., the observed data to condition on. 
+        x {torch.Tensor} -- conditioning variable for the posterior $p(\theta|x)$
     
     Keyword Arguments:
         num_samples {int} -- number of sample to generate (default: {1})
-        patience {int} -- upper time bound in minutes, in case sampling takes too long due to strong leakage (default: {5})
+        patience {int} -- upper time bound in minutes, in case sampling takes too long
+         due to strong leakage (default: {5})
     
     Returns:
-        Tuple[torch.Tensor, float] -- Accepted samples, and estimated acceptance probability
+        Tuple[torch.Tensor, float] -- Accepted samples, and estimated acceptance
+         probability
```

Comment:
Sorry, these are my old docstrings with types and Arguments etc. Could you please remove the types and replace with `Args`?

Reply:
No worries, I'll fix it.